api/rpc/account/account.go

```diff
@@ -186,6 +186,49 @@ func (s *Server) Login(ctx context.Context, in *LogInRequest) (*LogInReply, erro
 	return &LogInReply{Auth: token}, nil //Angular issue with custom header
 }
 
+// ResendVerify implements account.ResendVerify
+func (s *Server) ResendVerify(ctx context.Context, in *ResendVerifyRequest) (*empty.Empty, error) {
+	// Check an unverified user with the given email exists
+	users, err := s.Accounts.ListUsers(ctx)
+	if err != nil {
+		return nil, convertError(err)
+	}
+	unverifiedUserName := ""
+	for _, user := range users {
+		if user.Name == in.Name && !user.IsVerified {
+			unverifiedUserName = user.Name
+		}
+	}
+	if unverifiedUserName == "" {
+		return nil, convertError(fmt.Errorf("No unverified users with the given email"))
+	}
+
+	// Get User
+	unverifiedUser, err := s.Accounts.GetUser(ctx, unverifiedUserName)
+	if err != nil {
+		return nil, convertError(err)
+	}
+
+	// Create a verification token valid for an hour
+	token, err := s.Tokens.CreateVerificationToken(unverifiedUser.Name)
+	if err != nil {
+		if errd := s.Accounts.DeleteNotVerifiedUser(ctx, unverifiedUser.Name); errd != nil {
```

Comment:
I don't think you want to delete the user just because of a failed verification email resent